core/input.ts

```diff
@@ -75,7 +75,7 @@ export class Input {
    *     field again.  Should be unique to the host block.
    * @returns The input being append to (to allow chaining).
    */
-  appendField(field: string|Field, opt_name?: string): Input {
+  appendField(field: string|Field<any>, opt_name?: string): Input {
```

Comment:
I think this should be changed to a generic method, rather than `<any>` because `any`. My understanding is that using a generic method (`<T>`) means that the type is infered (external devs don't have to cast using `as Field`), but we still get the safety of using `unknown`.

We can convert the type to `Field` internally at the point we `splice` the field into the `fieldRow`.

Reply:
Done!